src/components/WorkInProgress/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import ExternalLink from 'components/ExternalLink';
+
+const WorkInProgress = ({ isClipped }) => (
+  isClipped
```

Comment:
what if the `defaultMessage` value was a conditional variable instead of the whole `<FormattedMessage>` component since thats the only part that gets clipped?

Reply:
You'll see the id's are different too so we can define two distinct translations. So really the defaultMessage is different, but so are each of the translations for each language. This also allows the 'extract-react-intl-messages' package to grabs these definitions independently to define to separate translations in our translated content json files